### DISCUSSION TITLE
my project and knowledge

### DISCUSSION THREAD

**Discussion by monesh48** [community/community#68204] [Programming Help]
hi guys,
   this is monesh from DI studios
    I'm interested game development, and i 'm searching programmers for my first game project. I had some basic knowledge full stack development ,but not a very good developer.so i need team who teach me as well as work with me in this project.

**Comment by emullallphotography** [comment]
Hi @monesh48,

I would recommend to use Unreal Engine with C++ because C++ is a powerful and fast programming language which makes it used most for game development and Unreal Engine for the default game engine. 

The documentation is here: https://www.unrealengine.com/en-US.

But the problem is I am not good enough at it to help you. Sorry!

Or if you want to use Pygame for really basic to semi complex games than I can collaborate with you for free
